### ISSUE TITLE
not entirely functional on XP home SP2?

### ISSUE THREAD

**Issue by pdemal** [Feodor2/Mypal#103]
Hello! I am happy to use Mypal as my main browser under XP Home SP2. Thank you very much!
Nevertheless, I have observed some issues:
On "https://lczero.org/", the button "Elo Estimates" does not work, when it ought display a graph.
On "https://pcsupport.lenovo.com/us/fr/products/laptops-and-netbooks/thinkpad-x-series-laptops/thinkpad-x230/downloads/driver-list/", no button works, for example a click on "Audio" gives no answer, when it ought initiate a download.
So Mypal seems not to be entirely functional. Is it a way to fix that?
Regards.

**Comment by Feodor2** [comment]
I shall examine that.
Certainly there are many other sites where Mypal not to be entirely functional. I consider that you may thank for this greed site owners who hire ignorant developers cheap.

**Comment by Feodor2** [comment]
So
1. https://lczero.org/ connects to the google stuff, and google oppresses non their browsers what would you expect. Try change user agent string for chrome
2. It is triggered on rectangle corners though, need further investigate.

**Comment by pdemal** [comment]
Hello,
I don't know if it can help, but on 
https://www.slimjet.com/en/dlpage_xp.php, you can find slimjet for xp 
(slimjet 10.0.13.0).
It works perfectly well in the two examples I gave.
Of course I should prefer to keep Mypal!
Thank you very much.

Feodor2 a écrit :

**Comment by ByRodrigoSoares** [comment]
My browsers are only MyPal and Maxthon

**Comment by ByRodrigoSoares** [comment]
I've used slimjet for XP was very good this browser over time and without updates many sites do not open anymore

**Comment by rereser** [comment]
general.useragent.override.lenovo.com
Mozilla/5.0 (Windows; WOW64; x64; rv:52.0) Gecko/20100101 Firefox/52.0

**Comment by pdemal** [comment]
I set a new entry in about:config: "general.useragent.override.lenovo.com" with the string "Mozilla/5.0 (Windows; WOW64; x64; rv:52.0) Gecko/20100101 Firefox/52.0" as indicated by rereser (thanks very much!). Now Mypal works fine in my second example.
Of course I keep the first example issue, but I observed that the last Firefox (unlike Chromium) has the same issue. So no hope this can be fixed in Mypal.
